cortex-m-pac/Cargo.toml

```
@@ -0,0 +1,24 @@
+[package]
+name = "cortex-m-pac"
+version = "0.1.0"
+edition = "2021"
+rust-version = "1.60"
```

```suggestion
rust-version = "1.81"
```

due to `core::error::Error`